x/concentrated-liquidity/incentives_test.go

```diff
@@ -1753,6 +1753,12 @@ func (s *KeeperTestSuite) TestInitOrUpdatePositionUptime() {
 					// 100 + 103 + UGI = 303
 					sdk.NewDecCoin("foo", sdk.NewInt(303)),
 				),
+				sdk.NewDecCoins(
+					// 100 + 112 + UGI = 312
+					sdk.NewDecCoin("bar", sdk.NewInt(309)),
+					// 100 + 104 + UGI = 304
+					sdk.NewDecCoin("foo", sdk.NewInt(303)),
```

Comment:
The values do not match the comment

Reply:
Ah nice catch – it turns out it doesn't matter for this test case, but I triple-checked that the test performs as expected. Also fixed in most recent commit.